extensions/wikia/WikiaMaps/WikiaMaps.i18n.php

```diff
@@ -42,7 +42,6 @@
 	'wikia-interactive-maps-create-map-header' => 'Create a Map',
 	'wikia-interactive-maps-create-map-next-btn' => 'Next',
 	'wikia-interactive-maps-create-map-back-btn' => 'Back',
-	'wikia-interactive-maps-create-map-choose-type-geo' => 'Real Map',
 	'wikia-interactive-maps-create-map-choose-type-custom' => 'Custom Map',
 	'wikia-interactive-maps-create-map-title-placeholder' => 'e.g. Weapon Location in Skyrim',
 	'wikia-interactive-maps-create-map-browse-tile-set' => 'Browse existing tile sets',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikia-interactive-maps-create-map-browse-tile-set" 15 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)